v0/destinations/fb_custom_audience/transform.js

```diff
@@ -323,4 +356,34 @@ const process = event => {
   return processEvent(event.message, event.destination);
 };
 
-exports.process = process;
+const processRouterDest = async inputs => {
+  if (!Array.isArray(inputs) || inputs.length <= 0) {
+    const respEvents = getErrorRespEvents(null, 400, "Invalid event array");
+    return [respEvents];
+  }
+
+  const respList = await Promise.all(
+    inputs.map(async input => {
+      try {
+        return getSuccessRespEvents(
+          await process(input),
+          [input.metadata],
+          input.destination
+        );
```

Comment:
As `process` returns an array of transformed events
```suggestion
        const transformedList = await process(input);
        const responseList = transformedList.map(transformedPayload => {
          getSuccessRespEvents(
            transformedPayload,
            [input.metadata],
            input.destination
          );
        });
        return responseList;
```